libs/blocks/global-navigation/global-navigation.js

```diff
@@ -760,8 +760,9 @@ class Gnav {
     if (!link) return '';
 
     // Check which elements should be rendered
+    const isBrandImage = rawBlock.matches('.brand-image');
```

Comment:
Should we keep the class name while authoring to brand-image-only?

Reply:
Done